targets/ChibiOS/RP_PICO_W_RP2040/target_stubs.c

```
@@ -0,0 +1,18 @@
+//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include <stdbool.h>
+
+// Stub implementations for functions not available when GPIO API is disabled.
+
+__attribute__((weak)) bool CPU_GPIO_Initialize()
```

Cppcheck (2.20.0)

[style] The function 'CPU_GPIO_Initialize' is never used.

(unusedFunction)